modules/text_server_adv/text_server_adv.cpp

```diff
@@ -4903,28 +4903,28 @@ void TextServerAdvanced::_insert_num_systems_lang() {
 	// Persian and Urdu numerals.
 	{
 		NumSystemData pr;
-		pr.lang.insert(StringName("fa")); // Persian
-		pr.lang.insert(StringName("fa_AF"));
-		pr.lang.insert(StringName("fa_IR"));
-		pr.lang.insert(StringName("ks")); // Kashmiri
-		pr.lang.insert(StringName("ks_IN"));
-		pr.lang.insert(StringName("ks_Arab"));
-		pr.lang.insert(StringName("ks_Arab_IN"));
-		pr.lang.insert(StringName("lrc")); // Northern Luri
-		pr.lang.insert(StringName("lrc_IQ"));
-		pr.lang.insert(StringName("lrc_IR"));
-		pr.lang.insert(StringName("mzn")); // Mazanderani
-		pr.lang.insert(StringName("mzn_IR"));
-		pr.lang.insert(StringName("pa_PK")); // Panjabi
-		pr.lang.insert(StringName("pa_Arab"));
-		pr.lang.insert(StringName("pa_Arab_PK"));
-		pr.lang.insert(StringName("ps")); // Pushto
-		pr.lang.insert(StringName("ps_AF"));
-		pr.lang.insert(StringName("ps_PK"));
-		pr.lang.insert(StringName("ur_IN")); // Urdu
-		pr.lang.insert(StringName("uz_AF")); // Uzbek
-		pr.lang.insert(StringName("uz_Arab"));
-		pr.lang.insert(StringName("uz_Arab_AF"));
+		pr.lang.insert(SNAME("fa")); // Persian
+		pr.lang.insert(SNAME("fa_AF"));
+		pr.lang.insert(SNAME("fa_IR"));
+		pr.lang.insert(SNAME("ks")); // Kashmiri
+		pr.lang.insert(SNAME("ks_IN"));
+		pr.lang.insert(SNAME("ks_Arab"));
+		pr.lang.insert(SNAME("ks_Arab_IN"));
+		pr.lang.insert(SNAME("lrc")); // Northern Luri
+		pr.lang.insert(SNAME("lrc_IQ"));
+		pr.lang.insert(SNAME("lrc_IR"));
+		pr.lang.insert(SNAME("mzn")); // Mazanderani
+		pr.lang.insert(SNAME("mzn_IR"));
+		pr.lang.insert(SNAME("pa_PK")); // Panjabi
+		pr.lang.insert(SNAME("pa_Arab"));
+		pr.lang.insert(SNAME("pa_Arab_PK"));
+		pr.lang.insert(SNAME("ps")); // Pushto
+		pr.lang.insert(SNAME("ps_AF"));
+		pr.lang.insert(SNAME("ps_PK"));
+		pr.lang.insert(SNAME("ur_IN")); // Urdu
+		pr.lang.insert(SNAME("uz_AF")); // Uzbek
+		pr.lang.insert(SNAME("uz_Arab"));
+		pr.lang.insert(SNAME("uz_Arab_AF"));
```

Comment:
Not sure if it's useful here, this code (and other similar parts in TextServer) is called only once.